exercises/binary-search/introduction.md

```diff
@@ -0,0 +1,11 @@
+# Introduction
+
+You have stumbled upon a group of mathematicians who are also singer-songwriters.
+
+They have written songs for all their favorite numbers, and, as you can imagine, they have a lot of favorite numbers.
+
+You are curious to hear one of their songs, but with so many numbers to choose from, finding the right song could take a while.
+
+Fortunately, they have organized their songs in a playlist sorted by the title—which is simply the number that the song is about.
```

Comment:
The `—` is a special character, right? I find it somewhat awkward to read. To me it renders as "title-which", which looks odd.

Reply:
It's a long dash - we previously agreed (slash I insisted 😉) that we put a space either side of it, else I find it illegible too, despite the fact the space is optional so it's technically correct (and in fact commonly used)